src/ray/raylet_client/raylet_client.cc

```diff
@@ -512,6 +512,16 @@ void raylet::RayletClient::ShutdownRaylet(
   grpc_client_->ShutdownRaylet(request, callback);
 }
 
+void raylet::RayletClient::DrainRaylet(
```

Comment:
Maybe we should sync with the prod team if timeout is necessary or not?

Reply:
Discussed offline: caller should add timeout if they want, we don't need to do anything here.